### PR TITLE
Add rejsekort.dk password change url and rules

### PR DESCRIPTION
**Pull request by MarkHjorth**
Screenshot showing change password url and password rules.
Screen Shot 2020-06-10 at 20 10 30

Rules tested with the Password Rules Validation Tool

Two commits, because I forgot to sort the list alphabetically 😅

### Checklist
- [x] The PR isn't documenting something that would be a common practice among password managers (e.g. minimal length of 6)
- [x] The top-level JSON objects are sorted alphabetically 
- [x] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update.

#### for change-password-URLs.json
- [x] There is no Well-Known URL for Changing Passwords (`https://example.com/.well-known/change-password`)

### CODE REVIEW

**Comment by rmondello** [comment]
If the change I suggests looks good to you and you update the PR, I’d be happy to merge it! :)

File: quirks/password-rules.json [hunk 1]
```diff
@@ -213,6 +213,9 @@
     "propelfuels.com": {
         "password-rules": "minlength: 6; maxlength: 16;"
     },
+    "rejsekort.dk": {
+        "password-rules": "minlength: 7; maxlength: 15; required: lower, upper, digit; allowed: lower, upper, digit;"
```
[the remark this answers is not in the document]
Comment (MarkHjorth) [reply quirks/password-rules.json:217]: Okay, I'll remove the allowed part.

What would the difference be between these two:

```
required: lower, upper; required: digit;
```
and
```
required: lower, upper, digit;
```

[the remark this answers is not in the document]
Comment (rmondello) [reply quirks/password-rules.json:217]: ```required: lower, upper; required: digit;```

A lowercase or uppercase character is required. A digit is required.

```required: lower, upper, digit;```

A lowercase or uppercase or digit character is required.

[the remark this answers is not in the document]
Comment (MarkHjorth) [reply quirks/password-rules.json]: I see.
I've updated the password rules so they adhere to the websites requirements (at least 1 uppercase, 1 lowercase and 1 digit)

**Comment by MarkHjorth** [comment]
Bump

**Comment by rmondello** [comment]
Thanks @MarkHjorth!

**Pull request merged**